### ISSUE TITLE
429 on any request

### ISSUE THREAD

**Issue by StNekroman** [LarryDpk/Google-Bard#26]
Any request gives me 429 http error with redirect link to google's page "Are you human? We suspect, you're robot" + capcha

Did logout in browser on Bard page, picked new cookies - the same.

**Comment by LarryDpk** [comment]
oh. Sorry to be late. did you frequently make a lot of call? It's still ok for me

**Comment by StNekroman** [comment]
Yes, very frequently - 5 requests in 10 minutes and I become kicked out.

**Comment by LarryDpk** [comment]
Google Bard may identify and ban.
